xesar-connect/src/main/kotlin/com/open200/xesar/connect/messages/query/OfficeMode.kt

```diff
@@ -0,0 +1,62 @@
+package com.open200.xesar.connect.messages.query
+
+import QueryElementResource
+import QueryListResource
+import com.open200.xesar.connect.utils.LocalTimeSerializer
+import com.open200.xesar.connect.utils.UUIDSerializer
+import java.time.LocalTime
+import java.util.*
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents an office mode in the system.
+ *
+ * @param id The unique identifier of the office mode.
```

Comment:
For strange reasons the `id` is the installation point ID in this case, like the `installationPointId` property.